src/Microsoft.VisualStudio.Services.Agent/HostTraceListener.cs

```diff
@@ -48,8 +50,9 @@ public HostTraceListener(string logFile)
             : base()
         {
             ArgUtil.NotNullOrEmpty(logFile, nameof(logFile));
-            Directory.CreateDirectory(Path.GetDirectoryName(logFile));
-            Stream logStream = new FileStream(logFile, FileMode.Create, FileAccess.ReadWrite, FileShare.Read, bufferSize: 4096);
+            _logFilePath = logFile;
```

Comment:
Why are we changing to store as instance var?

Reply:
so that in the event that we need to report to the user to look at the log, we can print out the log file path